src/main/java/com/wavefront/sdk/common/Utils.java

```diff
@@ -205,6 +211,12 @@ public static String tracingSpanToLineData(String name, long startMillis, long d
         sb.append(sanitize(tag._2));
       }
     }
+    if (spanLogs != null  && !spanLogs.isEmpty()) {
+      sb.append(' ');
+      sb.append(SPAN_LOG_KEY);
```

Comment:
needs to be wrapped in quotes (i.e. `"_spanLogs"="true"`), else proxy will reject the span

Reply:
Wrap in `sanitize` and you should be good.

Reply:
fixed